rest/blip_sync.go

```diff
@@ -128,7 +129,7 @@ func (ctx *blipSyncContext) register(profile string, handlerFn func(*blipHandler
 
 	ctx.blipContext.HandlerForProfile[profile] = func(rq *blip.Message) {
 
-		base.LogTo("Sync", "%s %q ... %s", rq, profile, ctx.effectiveUsername)
+		base.LogTo("Sync", "[%s] %s %q ... %s", ctx.blipContext.ID, rq, profile, ctx.effectiveUsername)
```

Comment:
I don't think we should be prepending the context ID manually on every LogTo call - it makes it too difficult to maintain consistent usage.  

At minimum we should add a helper method (or methods) in blip_sync.go that's used for all context-based logging.

Reply:
Agreed, I was hesitant to manipulate format string + args, but I guess having this duplicated all over the place is worse.

> At minimum we should add a helper method (or methods) in blip_sync.go that's used for all context-based logging.

Done -- see latest commits